Gopkg.toml

```diff
@@ -14,9 +14,21 @@
   version = "kubernetes-1.14.0"
   name = "k8s.io/apimachinery"
 
+[[override]]
+  version = "kubernetes-1.14.0"
+  name = "k8s.io/apiserver"
+
+[[override]]
+  version = "kubernetes-1.14.0"
+  name = "k8s.io/cli-runtime"
+
+[[override]]
+  version = "kubernetes-1.14.0"
+  name = "k8s.io/apiextensions-apiserver"
+
 [[constraint]]
   name = "k8s.io/kubernetes"
-  version = "v1.14.0"
+  version = "=v1.14.0"
```

Comment:
I have to read through. whats this `=` does here? limiting to exact version?

Reply:
yes, it will lock  to exact  version, I have faced a lot of issues (i thought  fixing to exact  version is a better option)